### PR TITLE
Added Docker cluster tutorial

### PR DESCRIPTION
**Pull request by yuce**
## Overview

[Describe what this pull request addresses.]

Fixes #

## Pull request checklist

- [ ] I have read the [contributing guide](https://github.com/pilosa/pilosa/blob/master/CONTRIBUTING.md).
- [ ] I have agreed to the [Contributor License Agreement](https://cla-assistant.io/pilosa/pilosa).
- [ ] I have updated the [documentation](https://github.com/pilosa/pilosa/tree/master/docs).
- [ ] I have resolved any merge conflicts.
- [ ] I have included tests that cover my changes.
- [ ] All new and existing tests pass.

## Code review checklist
This is the checklist that the reviewer will follow while reviewing your pull request. You do not need to do anything with this checklist, but be aware of what the reviewer will be looking for.

- [ ] Ensure that any changes to external docs have been included in this pull request.
- [ ] If the changes require that minor/major versions need to be updated, tag the PR appropriately.
- [ ] Ensure the new code is [properly commented](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments) and follows [Idiomatic Go](https://dmitri.shuralyov.com/idiomatic-go).
- [ ] Check that tests have been written and that they cover the new functionality.
- [ ] Run tests and ensure they pass.
- [ ] Build and run the code, performing any applicable integration testing.

### CODE REVIEW

File: docs/tutorials.md [hunk 1]
```diff
@@ -238,6 +238,92 @@ curl -k --ipv4 https://02.pilosa.local:10502/index/sample-index/query -d 'Bitmap
...
+
+And similarly for the second node:
+```
+curl localhost:10102/status
+```
+
+Outputs:
+
+```
+{"state":"NORMAL","nodes":[{"id":"123146fd-f25a-414b-9eb7-4619e8001858","uri":{"scheme":"http","host":"pilosa1","port":10101},"isCoordinator":true},{"id":"4dc45f41-6f9c-4417-ad0e-67522bfeef89","uri":{"scheme":"http","host":"pilosa2","port":10101},"isCoordinator":false}]}
```
Comment (travisturner) [inline docs/tutorials.md]: This response (even though it's just an example) should be the same as the response from the first node.

File: docs/tutorials.md [hunk 2]
```diff
@@ -238,6 +238,92 @@ curl -k --ipv4 https://02.pilosa.local:10502/index/sample-index/query -d 'Bitmap
...
+docker run -it --rm --name pilosa1 -p 10101:10101 --network="pilosanet" pilosa/pilosa:latest server --bind pilosa1 --cluster.coordinator=true --gossip.seeds=pilosa1:14000
+```
+
+Let's run the second Pilosa node and attach it to the virtual network as well. Note that we set the address of the gossip seed to the address of the first node:
+```
+docker run -it --rm --name pilosa2 -p 10102:10101 --network="pilosanet" pilosa/pilosa:latest server --bind pilosa2 --gossip.seeds=pilosa1:14000
+```
+
+Let's test that the nodes in the cluster connected with each other:
+```
```
Comment (travisturner) [inline docs/tutorials.md]: can we use the request/response language tags here?

File: docs/tutorials.md [hunk 3]
```diff
@@ -238,6 +238,92 @@ curl -k --ipv4 https://02.pilosa.local:10502/index/sample-index/query -d 'Bitmap
 
 Check out our [Administration Guide](https://www.pilosa.com/docs/latest/administration/) to learn more about making the most of your Pilosa cluster and [Configuration Documentation](https://www.pilosa.com/docs/latest/configuration/) to see the available options to configure Pilosa.
 
+### Setting Up a Docker Cluster
+
+In this tutorial, we will be setting up a 2-node Pilosa cluster using Docker containers. The instructions below require Docker 1.13 or better.
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/tutorials.md]: We're referring to the Docker version. Docker changed their versioning scheme before, so 18.x is in fact 1.18.x.

File: docs/tutorials.md (hunk 1 above)
Comment (yuce) [reply docs/tutorials.md]: Fixed

File: docs/tutorials.md (hunk 2 above)
Comment (yuce) [reply docs/tutorials.md]: Updated.

Comment (yuce) [reply docs/tutorials.md]: Ok, updated.

**Pull request merged**